src/vfc_precexp

```diff
@@ -0,0 +1,514 @@
+#!/usr/bin/env python3
+import numpy as np
+import pandas as pd 
+import matplotlib.pyplot as plt
+import random
+import math
+import sys
+import struct
+import shlex
+import os.path
+import subprocess
+import signal
+from os import listdir
+
+vfc_profile_file = "vfc_profile.txt"
+vfc_config_file = "vfc_config.txt"
+output_dir = ["vfc_ref", "vfc_std"]
+
+def set_environment_variable(envVar, value, env):
+	#replace current value, if the environment variable already exists
+	env[envVar.strip()] = value.strip()
+
+	return env
+
+def unset_environment_variable(envVar, env):
+	#replace current value, if the environment variable already exists
+	del env[envVar.strip()]
+
+	return env
+
+
+def add_environment_variable(envVar, value, env):
+	#check if the environment variable already exists, then append or add the new value
+	if (envVar.strip() in env):
+		env[envVar.strip()] = value + ":" + env[envVar.strip()]
```

Comment:
I think you are missing a `.strip()` after `value` here.

Reply:
it's fixed